tests/include/common.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
-NFT=`which nft`
+NFT=/usr/sbin/nft
```

Comment:
this change feels kind of bad but as-is it was the path on the host which was different from the path in the container